apis/env.d.ts

```diff
@@ -7,7 +7,12 @@
 export const env: {
   build: any,
   hana: any,
-  i18n: any,
+  i18n: {
+    languages: [string],
+    default_language: string,
+    folders: [string],
```

Comment:
are these meant to be string arrays of arbitrary length? If so, then we should `string[]` instead (`[string]` is a tuple with exactly one string element)

Reply:
Good catch, accidentally wrote this in Swift syntax here 😃